TimberWinR/Outputs/Redis.cs

```diff
@@ -79,6 +174,7 @@ public override JObject ToJson()
                     new JObject(
                         new JProperty("host", string.Join(",", _redisHosts)),
                         new JProperty("errors", _errorCount),
+                        new JProperty("lastErrorTimeUTC", _lastErrorTimeUTC),
```

Comment:
There seems to be a mixture of snake case and camel case variables here. Since this is json and all of the additions are camel case, I think the right thing to do would be to snake_case the additions.